pilot/docker/Dockerfile.pilot

```diff
@@ -15,7 +15,6 @@ FROM docker.io/istio/distroless:${BASE_VERSION} as distroless
 FROM ${BASE_DISTRIBUTION}
 
 COPY pilot-discovery /usr/local/bin/
-COPY cacert.pem /cacert.pem
```

Comment:
can you also update the Makefile so that the pilot docker target doesn't depend on the cacert.pem?